src/extension/index.ts

```diff
@@ -109,6 +111,52 @@ function activateDiagnostics(disposables: Disposable[], store: Store, baser: Uri
     disposables.push(workspace.onDidCloseTextDocument(doc => diagsAll.delete(doc.uri))); // Spurious *.git deletes don't hurt.
     const disposer = observe(store.logs, () => workspace.textDocuments.forEach(setDiags));
     disposables.push({ dispose: disposer });
+    return setDiags;
+}
+
+function activateCodeActions(disposables: Disposable[], baser: UriRebaser, refreshDiagnostics: (document: TextDocument) => void) {
```

Comment:
Just my opinion, but having 2 functions nested inside this larger function makes this very difficult for me to read :(. I would assume that the "fix function" is going to get a lot bigger, so why not split it out into another function or even another file?

Reply:
Discussed offline. In short there's some question to the viability of this feature without significant additional investment in "tracking spans". Thus keeping these lines of code "packed-up and ready to go" just in case.